### PR TITLE
Restored `-v` to display Newman version

### PR DESCRIPTION
**Pull request by kunagpal**

### CODE REVIEW

File: lib/cli/index.js [hunk 1]
```diff
@@ -237,10 +237,6 @@ var _ = require('lodash'),
             addHelp: false
         });
 
-        commonParser.addArgument(['-v', '--version'], {
```
Comment (shamasis) [inline lib/cli/index.js]: @czardoz were these duplicates? How!

File: lib/cli/index.js [hunk 2]
```diff
@@ -384,13 +380,6 @@ var _ = require('lodash'),
             prog: program
         });
 
-        parser.addArgument(['--version'], {
```
Comment (shamasis) [inline lib/cli/index.js]: @czardoz were these duplicates? How!

**Comment by shamasis** [comment]
LGTM. @czardoz - review and merge please.

File: lib/cli/index.js [hunk 3]
```diff
@@ -384,13 +380,6 @@ var _ = require('lodash'),
...
-            help: 'Prints the version and exits',
```
Comment (shamasis) [inline lib/cli/index.js]: As per discussion with @czardoz we should keep the duplicates so that help text show up for legacy help text.

Comment (kunagpal) [inline lib/cli/index.js]: Okay.

**Pull request merged**